Example/Sources/View Controllers/AdvancedExampleViewController.swift

```diff
@@ -311,6 +316,14 @@ extension AdvancedExampleViewController: MessagesDisplayDelegate {
         avatarView.layer.borderColor = UIColor.primaryColor.cgColor
     }
     
+    func configureAccessoryView(_ accessoryView: UIView, for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) {
+        let button = UIButton(type: .infoLight)
+        button.tintColor = .primaryColor
+        accessoryView.addSubview(button)
```

Comment:
This subview will have to be removed somewhere because cells are reused?

Reply:
Added with a comment